index.d.ts

```diff
@@ -238,7 +300,13 @@ interface Taggable {
 }
 
 interface StartSpanFn {
-  startSpan (name?: string, type?: string, options?: SpanOptions): Span | null;
+  startSpan (
```

Comment:
I think this interface should be the same as the `startSpan` definitions defined in `Transaction` and `Agent`

Reply:
fixed